components/input-mapper/pom.xml

```diff
@@ -22,7 +22,7 @@
     <parent>
         <groupId>org.wso2.extension.siddhi</groupId>
         <artifactId>siddhi-map-json</artifactId>
-        <version>4.0.0-M5-SNAPSHOT</version>
```

Comment:
it should be component/pom.xml  rename to component remove input-mapper and output-mapper modules both need to be in one module